internal/core/experiment_run.go

```diff
@@ -239,22 +239,24 @@ func (b *ModelRegistryService) GetExperimentRunArtifacts(artifactType openapi.Ar
 
 func (b *ModelRegistryService) GetExperimentRunMetricHistory(name *string, stepIds *string, listOptions api.ListOptions, experimentRunId *string) (*openapi.MetricList, error) {
 
-	// Validate experiment run exists
-	if experimentRunId == nil {
-		return nil, fmt.Errorf("experiment run ID is required: %w", api.ErrBadRequest)
-	}
+	var experimentRunIdInt32Ptr *int32
 
-	// Validate experiment run exists
```

Comment:
Since this validation is removed, would we not get an error message if user gets metric history for a non-existent experiment run id? If so, it would cause failure in fuzzer tests.

Reply:
I was just wondering about the `/experiment_runs/32/metric_history` scenario. Whether there would be any url representation that would drop the id or set it to null somehow. But, I don't think there is a way, so this should be ok.